packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md

```diff
@@ -350,7 +341,7 @@ To share the certificate that you created across multiple apps, clone it with th
 
 To share the certificate that you created across multiple IdPs, clone it with the [IdPs API](#top) by using the key `id` that you generated.
 
-Be sure to clone the certificate to every app or IdP that you want to share it with. After it is cloned, you need to [update the key credential](/docs/guides/sign-your-own-saml-csr/update-the-key-credential/) for the target app.
+Be sure to clone the certificate to every app or IdP that you want to share it with. After it is cloned, you need to [update the key credential](#update-the-key-credential) for the target app.
```

Comment:
```suggestion
Ensure that you clone the certificate to every app or IdP that you want to share it with. After the certificate is cloned, you need to [update the key credential](#update-the-key-credential) for the target app.
```

Reply:
On line 352, remove the capital letters from the link: `[Publish a CSR with a certificate](#publish-a-csr-with-a-certificate)`

(sorry, I can't make this suggestion on the exact line in this review)